packages/tldraw/src/test/Editor.test.tsx

```diff
@@ -446,7 +446,9 @@ describe('isFocused', () => {
 		expect(editor.getInstanceState().isFocused).toBe(false)
 	})
 
-	it('becomes false when a child of the app container div receives a focusout event', () => {
+	it.skip('becomes false when a child of the app container div receives a focusout event', () => {
```

Comment:
hm, i dunno why this test needed to be paused actually. i stared at it but not seeing the difference. @SomeHats if you have more historical context here maybe lemme know.

Reply:
So in the past, if the tldraw editor was focused, and then focus left (e.g. went to some input somewhere else on the page) we would update the instance state appropriately. I honestly have no idea where we were/are doing this though, but the test still seems to make some conceptual sense to me?

Reply:
Searching for `focusout` it seems like we only used it in `useSafariFocusOutFix`, which has been completely removed in this PR. Not sure we even handle this event anywhere, so the test might just be obsolete?

Reply:
yeah, but the weird thing is, this test was testing not just Safari 🤔

Reply:
is this not generally desired behaviour though? if focus leaves the editor in the DOM, the internal state of the editor should reflect that, right?

Reply:
Well, to be clear though, this is leaving focus from a child of the editor, not the editor directly. I think if it leaves focus from a child it would just go to a parent element possibly? It's sort of depends on where the person clicks next or tabs to next.